Lesson15/elifbasak/script.js

```diff
@@ -0,0 +1,223 @@
+const y = 10;
+const z = 15;
+console.log("Ex.17.", t + y + z);
+console.log("---------------------------------");
+
+// 18. Declare dividend=10 and divisor=3. Log the quotient (divisionResult) and difference (differenceResult).
+// CODE HERE
+
+const dividend = 10;
+const divisor = 3;
+const divisionResult = dividend / divisor;
+const differenceResult = dividend - divisor;
+console.log(
+  "Ex.18.",
+  "Bölme sonucu:",
+  divisionResult,
+  "Çıkarma sonucu:",
+  differenceResult
+);
+console.log("---------------------------------");
+
+// 19. Declare firstName and lastName. Create fullName by concatenating them with a space and log it.
+// CODE HERE
+
+const firstName = "İdil";
+const lastName = "Doğa";
+const fullName = firstName + " " + lastName;
+console.log("Ex.19.", fullName);
+console.log("---------------------------------");
+
+// 20. Declare firstFactor=7 and secondFactor=2. Log the product.
+// CODE HERE
+
+const firstFactor = 7;
+const secondFactor = 2;
+console.log("Ex.20.", firstFactor * secondFactor);
+console.log("---------------------------------");
+
+// 21. Log the value of Math.PI.
+// CODE HERE
+
+console.log("Ex.21.", Math.PI);
+console.log("---------------------------------");
+
+// 22. Declare counter=0. Increment it using three different methods (e.g., counter++, counter+=1, counter=counter+1) and log the result each time.
+// CODE HERE
+
+let newCounter = 0;
+newCounter++;
+console.log("Ex.22.", newCounter);
+newCounter += 1;
+console.log("Ex.22.", newCounter);
+newCounter = newCounter + 1;
+console.log("Ex.22.", newCounter);
+console.log("---------------------------------");
+
+// 23. Declare initialTemperature=20. Increase it by 5 and log the result.
+// CODE HERE
+
+let initialTemperature = 20;
+initialTemperature += 5;
+console.log("Ex.23.", initialTemperature);
+console.log("---------------------------------");
+
+// 24. Declare numberEx9=6. Increment it using the prefix ++ operator and log both the variable and the incremented value.
+// CODE HERE
+
+let numberEx9 = 6;
+console.log("Ex.24.", "İlk değer:", numberEx9);
+++numberEx9;
+console.log("Ex.24.", "Arttirilmiş değer:", numberEx9);
+console.log("---------------------------------");
+
+// 25. Declare numberEx10=8. Increment it using the postfix ++ operator and log both the original variable and the incremented value.
+// CODE HERE
+
+let numberEx10 = 8;
+console.log("Ex.25.", "İlk değer:", numberEx10);
+numberEx10++;
+console.log("Ex.25.", "Arttirilmiş değer:", numberEx10);
+console.log("---------------------------------");
+
+// 26. Declare numberEx11=-3. Increment it by 1 using prefix ++, then multiply the result by 2 and log the final value.
+// CODE HERE
+
+let numberEx11 = -3;
+let finalValue = ++numberEx11 * 2;
+console.log("Ex.26.", finalValue);
+console.log("---------------------------------");
+
+// 27. Declare a=2 and b=3. Increment a using the prefix ++ operator, then add b to the result and log it.
+// CODE HERE
+
+let a1 = 2;
+let b1 = 3;
```

Comment:
The b1 variable does not have to be defined with let since its value never changes.